.github/workflows/configlet.yml

```diff
@@ -13,5 +13,3 @@ permissions:
 jobs:
   configlet:
     uses: exercism/github-actions/.github/workflows/configlet.yml@main
-    with:
-      fmt: true
```

Comment:
This should remain unchanged, as that is what the workflow does: https://github.com/exercism/csharp/blob/main/.github/workflows/configlet.yml

Reply:
Indeed